src/components/DocsHome/Languages/index.tsx

```diff
@@ -82,8 +83,23 @@ function Index() {
               Kybra allows you to build Internet Computer dapps and smart contracts using Python, one of the most popular languages in the world
             </p>
           </div>
-          <img className={styles.motokoBackground} src={kybraBG} alt=""/>
-          <RightArrowSVG className={styles.informationIcon}/>
+          <img className={styles.motokoBackground} src={kybraBG} alt="" />
+          <RightArrowSVG className={styles.informationIcon} />
+        </motion.a>
+        <motion.a
+          variants={transitions.item}
+          href={"https://bitfinity.network/ecosystem/solidity-examples"}
+          target={"_blank"}
+          className={clsx(styles.card, styles.cardHover)}
+        >
+          <div className={styles.bodyContainer}>
+            <p className={styles.informationTitle}>Solidity</p>
+            <p className={styles.informationBody}>
+              The Bitfinity EVM allows you to run Solidity smart-contracts on the Internet Computer
```

Comment:
```suggestion
              The Bitfinity EVM allows you to run Solidity smart contracts on the Internet Computer
```
according to the new style guide